File: examples/ruby/basic/03_collections.rb
# Ruby Basic: Collections

# Array
fruits = ["apple", "banana"]
fruits << "cherry" # append
puts "Fruits: #{fruits.inspect}"

# Hash (Key-Value)
person = { name: "Alice", age: 25 }
puts "Person: #{person[:name]}"

# Iterating
fruits.each do |fruit|
  puts "I like #{fruit}"
end

File: examples/rust/basic/03_collections.rs
// Rust (Rune) Basic: Collections

pub fn main() {
    // Lists
    let fruits = ["apple", "banana"];
    
    // Objects (HashMaps)
    let person = #{ "name": "Alice", "age": 25 };
    
    println!("Fruits: {:?}", fruits);
    println!("Person name: {}", person["name"]);
    
    for fruit in fruits {
        println!("Fruit: {}", fruit);
    }
}
